beta-pipelines/skylab/m3c/Config files/Config_file/config.yaml

```diff
@@ -0,0 +1,8 @@
+# Path to hisat3n
+hisat3n_dna_reference: /tmp/docker_m3c/reference/hg38
```

Comment:
@nikellepetrillo Should we replace this with the mapping yaml that's used in the updated WDL?

Reply:
good point! just updated it